.github/workflows/dotnet-core.yml

```diff
@@ -79,6 +79,22 @@ jobs:
         (cd ii && dotnet publish --runtime linux-x64 -c Release --self-contained true -o ../linux-x64)
         (cd win-x64 && zip -9r ../release/ii-cli-win-x64.zip .)
         (tar -zcvf ./release/ii-cli-linux-x64.tar.gz ./linux-x64)
+    - name: Test ii binary
+      run: |
+         mkdir ./testii &&
+         cp ./release/ii-cli-linux-x64.tar.gz ./testii/ &&
+         cd ./testii &&
+         tar -xvf ./ii-cli-linux-x64.tar.gz &&
+         cd .. &&
+         ./testii/linux-x64/ii --help &&
+
+         wget https://github.com/HicServices/BadMedicine.Dicom/releases/download/v0.0.14/baddicom-linux-x64-v0.0.14.tar.gz &&
+         tar -xvf ./baddicom-linux-x64-v0.0.14.tar.gz &&
+         ./linux-x64/BadDicom -s 100 ./testdicoms/ 10 10 CT &&
```

Comment:
we don't need the `&&` at the end of each line here. Starting the whole block with `|` means that newlines are preserved: https://yaml-multiline.info/

Reply:
Will it still report any nonzero exit codes as failure (for any step)?  I thought that was how you got linux to aggregate exit codes?

Reply:
I think in that case it's better to put `set -e` at the start of the script, which will cause an exit on the first non-zero return code. See [here](http://redsymbol.net/articles/unofficial-bash-strict-mode/).